### PR TITLE
Add airframe .post scripts on NuttX targets

### PR DESCRIPTION
**Pull request by jlecoeur**
**Describe problem solved by the proposed pull request**
On POSIX targets, it is possible to add a `.post` script that will be run at the end of the startup.
See for example `ROMFS/px4fmu_common/init.d-posix/1013_iris_vision.post`.

On Nuttx targets, this is not supported.

**Describe your preferred solution**
Add support for `.post` scripts on NuttX targets.

**Additional context**
Similar to how `ROMFS/px4fmu_common/init.d/rc.autostart` is generated at compile time, this PR generates a new file `ROMFS/px4fmu_common/init.d/rc.autostart.post` which is called at the end of `ROMFS/px4fmu_common/init.d/rcS`.

Test example:
- add a file `ROMFS/px4fmu_common/init.d/airframes/4001_quad_x.post`
- with the following content `mavlink stream -d /dev/ttyS1 -s DEBUG_VECT -r 2`
- add an entry `4001_quad_x.post` in `ROMFS/px4fmu_common/init.d/airframes/CMakeLists.txt`
- rebuild
- the default rate for the mavlink message DEBUG_VECT is now 2Hz instead of 1Hz

@bkueng this is what we discussed on Slack. By default the new script is emtpy so it does not increase the flash usage.
@hamishwillee this should not affect the autogenerated airframe mardown file, but better double check.

### CODE REVIEW

File: Tools/px4airframes/rcout.py [hunk 1]
```diff
@@ -41,7 +41,19 @@ def __init__(self, groups, board):
                         excluded = True
                 if excluded:
                     continue
-                path = os.path.split(param.GetPath())[1]
+
+                if not post_start:
```
Comment (bkueng) [inline Tools/px4airframes/rcout.py]: Can you simplify to:
```
                if post_start:
                    # Path to post-start script
                    path = param.GetPostPath()
                else:
                    # Path to start sript
                    path = param.GetPath()

                if not path:
                    continue
                path = os.path.split(post_path)[1]
```

File: Tools/px4airframes/srcparser.py [hunk 2]
```diff
@@ -141,6 +142,9 @@ def __init__(self, path, name, airframe_type, airframe_class, airframe_id, maint
     def GetPath(self):
         return self.path
 
+    def GetPostPath(self):
```
Comment (bkueng) [inline Tools/px4airframes/srcparser.py]: Maybe some docs would be useful here?

Comment (jlecoeur) [reply Tools/px4airframes/srcparser.py]: done!

File: Tools/px4airframes/rcout.py (hunk 1 above)
Comment (jlecoeur) [reply Tools/px4airframes/rcout.py]: thanks

**Comment by jlecoeur** [comment]
@bkueng Thanks for the review, I made the changes, can you review again?

> Looks good, I assume
> 
> https://github.com/PX4/Firmware/blob/70023eb263d4b2381b6f653910c5fc240febb3ca/Tools/px4airframes/srcparser.py#L269
> will filter the .post files, did you check that?

Yes, the .post files do not need to be parsed, so this is fine like this.

**Pull request merged**